dev-docs/modules/geolocationRtdProvider.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: page_v2
+title: Geolocation Module
+display_name: Geolocation
+description: Real Time Geolocation
+page_type: module
+module_type: rtd
+module_code: geolocationRtdProvider
+enable_download: true
+sidebarType: 1
+---
+
+# Geolocation Module
+
+## Overview
+
+The Geolocation module lets publishers get user's precise location with their permissions. The first permission that is needed is directly asked from site's navigator alert. Then if site has installed a CMP (Consent Management Platform), this module checks the geolocation permission from consent data. The module provides Geolocation coords using the [Geolocation API](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation_API).
+
+## Integration
+
+1. Build the geolocation RTD module into the Prebid.js package with:
+
+    ```bash
+    gulp build --modules=geolocationRtdProvider.js,...
+    ```
+
+2. Use `setConfig` to instruct Prebid.js to initilize the geolocation module, as specified below.
+
+## Configuration
+
+This module is configured as part of the `realTimeData.dataProviders` object:
+
+```javascript
+pbjs.setConfig({
+    realTimeData: {
+        dataProviders: [{
+            "name": "geolocation",
+            "waitForIt": true,
+            "params": {
+                "requestPermission": true
+            }
+        }]
+    }
+});
+```
+
+Parameter details:
+
+{: .table .table-bordered .table-striped }
+|Name |Type |Description |Notes |
+| :------------ | :------------ | :------------ |:------------ |
+|name | String | Real time data module name |Required, always 'geolocation' |
+|waitForIt | Boolean | |Required, always true |
+|params | Object | | |
+|params.requestPermission | Boolean | Customer permission |Required, always true  |
+
+The Geolocation is set in ortb2Fragments.global.device.geo in requestBidsObject and looks like this:
```

Comment:
Sorry, I still don't fully understand how a bid adapter would make use of this data. I see a couple of adapters that do the following:
```
mergeDeep(reqBidsConfigObj.ortb2Fragments.global, _ortb2);
```
Is that all they need to do and the data will wind up in device.geo?

Reply:
yes, also they can get it while building request from function callback

Reply:
Cool - can you update the doc with a specific reference for how bid adapters should consume this info?